eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/model/SubmittedVariant.java

```diff
@@ -232,71 +243,28 @@ public void setValidated(Boolean validated) {
-            return false;
-        }
-        if (supportedByEvidence != null ? !supportedByEvidence.equals(
-                variant.supportedByEvidence) : variant.supportedByEvidence != null) {
-            return false;
-        }
-        if (assemblyMatch != null ? !assemblyMatch.equals(variant.assemblyMatch) : variant.assemblyMatch != null) {
-            return false;
-        }
-        if (allelesMatch != null ? !allelesMatch.equals(variant.allelesMatch) : variant.allelesMatch != null) {
-            return false;
-        }
-        if (validated != null ? !validated.equals(variant.validated) : variant.validated != null) {
-            return false;
-        }
-        return true;
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        SubmittedVariant that = (SubmittedVariant) o;
+        return taxonomyAccession == that.taxonomyAccession &&
+                start == that.start &&
+                Objects.equals(referenceSequenceAccession, that.referenceSequenceAccession) &&
+                Objects.equals(projectAccession, that.projectAccession) &&
+                Objects.equals(contig, that.contig) &&
+                Objects.equals(referenceAllele, that.referenceAllele) &&
+                Objects.equals(alternateAllele, that.alternateAllele) &&
+                Objects.equals(clusteredVariantAccession, that.clusteredVariantAccession) &&
+                Objects.equals(supportedByEvidence, that.supportedByEvidence) &&
+                Objects.equals(assemblyMatch, that.assemblyMatch) &&
+                Objects.equals(allelesMatch, that.allelesMatch) &&
+                Objects.equals(validated, that.validated);
     }
 
     @Override
     public int hashCode() {
-        int result = referenceSequenceAccession.hashCode();
-        result = 31 * result + taxonomyAccession;
-        result = 31 * result + projectAccession.hashCode();
-        result = 31 * result + contig.hashCode();
-        result = 31 * result + (int) (start ^ (start >>> 32));
-        result = 31 * result + referenceAllele.hashCode();
-        result = 31 * result + alternateAllele.hashCode();
-        result = 31 * result + (clusteredVariantAccession != null ? clusteredVariantAccession.hashCode() : 0);
-        result = 31 * result + (supportedByEvidence != null ? supportedByEvidence.hashCode() : 0);
-        result = 31 * result + (assemblyMatch != null ? assemblyMatch.hashCode() : 0);
-        result = 31 * result + (allelesMatch != null ? allelesMatch.hashCode() : 0);
-        result = 31 * result + (validated != null ? validated.hashCode() : 0);
-        return result;
+        return Objects.hash(referenceSequenceAccession, taxonomyAccession, projectAccession, contig, start,
+                            referenceAllele, alternateAllele, clusteredVariantAccession, supportedByEvidence,
+                            assemblyMatch, allelesMatch, validated);
```

Comment:
Similar to previous comment, have you verified the equivalence of this hash against the one obtained with more verbose code?